.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,5 @@
-### All Submissions:
+DO NOT CREATE NEW PULL REQUESTS ON THIS REPOSITORY!
 
-- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
-- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?
+Please create formula pull requests at https://github.com/Homebrew/homebrew-core
```

Comment:
People should be reading this in the new pull request where it will not be formatted.